src/index.js

```diff
@@ -1,20 +1,22 @@
-import _ from 'lodash';
-import printMe from './print.js';
+// import _ from 'lodash';
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻